lib/resolver.content.js

```diff
@@ -140,6 +142,22 @@ export default class PodletClientContentResolver {
             method: 'GET',
             query: outgoing.reqOptions.query,
             headers,
+            onInfo({ statusCode, headers }) {
+                if (statusCode === 103) {
+                    const parsedAssetObjects = parseLinkHeaders(headers.link);
+
+                    const scriptObjects = parsedAssetObjects.filter(
+                        (asset) => asset instanceof AssetJs,
+                    );
+                    const styleObjects = parsedAssetObjects.filter(
+                        (asset) => asset instanceof AssetCss,
+                    );
+                    // set the content js asset objects
+                    outgoing.js = filterAssets('content', scriptObjects);
+                    // set the content css asset objects
+                    outgoing.css = filterAssets('content', styleObjects);
+                }
+            },
```

Comment:
this onInfo block is called when early hints are received from the podlet during a content route request. AssetJs and AssetCss objects are created from the link header and then set on outgoing.js and outgoing.css

Reply:
I think we might need some kinda guard that only accepts one `onInfo` and `103` http event here.